Dockerfile.test

```diff
@@ -0,0 +1,25 @@
+FROM python:3.5
+
+WORKDIR /opt/fairing
+
+# Install dependencies
+COPY setup.py requirements.txt /opt/fairing/
+RUN pip install -e .
+
+# Install gcloud
+RUN mkdir -p /builder && \
+	wget -qO- https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz | tar zxv -C /builder && \
+	CLOUDSDK_PYTHON="python2.7" /builder/google-cloud-sdk/install.sh --usage-reporting=false \
```

Comment:
- python 3.5?
- pin to a specific gcloud version? 
- move this to the very top of the dockerfile, after WORKDIR

Dockerfile outline:
1. install gcloud
2. install pytest
3. copy setup.py and requirements, run pip install
4. copy over source
5. install fairing

Reply:
Updated